README.md

```diff
@@ -239,6 +239,61 @@ Create a replication stream. Options include:
 }
 ```
 
+## Replicating over network with hyperdiscovery
+
+In order to have your hyperdb replicate to remote peers, you must use [hyperdiscovery](https://github.com/karissa/hyperdiscovery). hyperdiscovery handles all of the peer discovery, replication, and streaming so that you don't have to.
+
+Once a peer is discovered and connected to, you must authorize that peer so that they can write changes to your hyperdb, otherwise replication will only happen one way.
```

Comment:
I feel like "replicating over the network" is a different usecase than "making a db writable to everyone on the network." The latter is what this example is doing!

Reply:
Noted, I have tried to clarify this a bit in my changes.